mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlSource.java

```diff
@@ -81,7 +81,7 @@ protected LineageRecorder getLineageRecorder(BatchSourceContext context) {
 
   @Override
   protected SchemaReader getSchemaReader() {
-    return new MysqlSchemaReader(null);
+    return new MysqlSchemaReader(null, mysqlSourceConfig.getConnectionArguments());
```

Comment:
This change is also needed here in `MysqlConnector`: https://github.com/data-integrations/database-plugins/blob/9ad84e9f50db276d87ce0f6a250576ac4998f4c7/mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlConnector.java#L65